utils/config_file.cc

```diff
@@ -221,6 +221,16 @@ sstring utils::hyphenate(const std::string_view& v) {
     return result;
 }
 
+sstring utils::trim_to_first_comma(const std::string_view& v) {
+    auto idx = v.find(',');
+    if (idx == 0) {
```

Comment:
find() returns npos, not 0, when a match is not found (https://en.cppreference.com/w/cpp/string/basic_string_view/find)